### DISCUSSION TITLE
Serverless Devs VsCode 插件

### DISCUSSION THREAD

**Discussion by heimanba** [Serverless-Devs/Serverless-Devs#264] [Ideas]
### 调研
1. midway 新建函数
![image](https://user-images.githubusercontent.com/5129967/138984933-5b35466a-9e27-484c-93b8-25915b4f2600.png)
2. tencent-scf
![image](https://user-images.githubusercontent.com/5129967/138985671-d0120cac-83ef-4298-934d-291f4f93efb2.png)
![image](https://user-images.githubusercontent.com/5129967/138985781-4662e18b-dd68-4502-aada-65292c56ccc6.png)
![image](https://user-images.githubusercontent.com/5129967/138985874-883fdf55-7db3-4ab4-8598-b74cbedf2541.png)
![image](https://user-images.githubusercontent.com/5129967/138986056-9780d2c2-511a-472a-89cc-3fc57c9242d0.png)
![image](https://user-images.githubusercontent.com/5129967/138986276-ce77aa9c-689f-48ac-956e-d11972300f23.png)
![image](https://user-images.githubusercontent.com/5129967/138986307-424cc112-d065-4eaf-b680-523286feec58.png)

### 本地开发
- 函数创建
![image](https://user-images.githubusercontent.com/5129967/138918587-9cec7ebf-0621-48ca-98e6-d2cbe220bcb3.png)
- 函数部署
![image](https://user-images.githubusercontent.com/5129967/138918676-574ec9f6-92c3-4d22-b3a0-20be23d8722d.png)

**Comment by anycodes** [comment]
这个应该是按照funcraft的那个插件模式做的。

**Reply by heimanba** [reply]
fun做的还不错，但是有点复杂，先调研下

**Reply by heimanba** [reply]
想先出一个MVP的版本，看下最小功能能做哪些
